sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScanBase.scala

```diff
@@ -1224,7 +1224,7 @@ private case class GpuOrcFileFilterHandler(
       val searchArg = readerOpts.getSearchArgument
       if (searchArg != null && orcReader.getRowIndexStride != 0) {
         val sa = new SargApplier(searchArg, orcReader.getRowIndexStride, evolution,
-          orcReader.getWriterVersion, useUTCTimestamp)
+          orcReader.getWriterVersion, useUTCTimestamp, false, false)
```

Comment:
Actually on second thought, we shouldn't be passing `false, false` here.  We should be following [what ORC 1.5.10 is doing here](https://github.com/apache/orc/blob/rel/release-1.5.10/java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java#L219-L220) which is looking up the parameters from the reader and options.

Reply:
Updated and added test case.